cascading-hadoop3-tez/src/test/java/cascading/stats/tez/TezStatsPlatformTest.java

```diff
@@ -59,7 +60,7 @@ public TezStatsPlatformTest()
     super( true, 1, 4 );
     }
 
-  @Test
+    @Ignore @Test
   public void testStatsCounters() throws Exception
     {
     getPlatform().copyFromLocal( inputFileApache );
```

Comment:
test using FlowStats API

Reply:
I realized disabling this test completely is wrong. I turned it back on but commented out the stats asserts after it runs since that requires the upstream TEZ changes.